redisinsight/ui/src/pages/home/components/databases-list/components/DatabasesListCellControls/methods/handlers.spec.ts

```diff
@@ -0,0 +1,134 @@
+import { TelemetryEvent, sendEventTelemetry } from 'uiSrc/telemetry'
+import { dispatch } from 'uiSrc/slices/store'
+import { Instance } from 'uiSrc/slices/interfaces'
+
+import {
+  handleDeleteInstances,
+  handleClickDeleteInstance,
+  handleClickGoToCloud,
+  handleClickEditInstance,
+  handleManageInstanceTags,
+} from './handlers'
+
+jest.mock('uiSrc/slices/store', () => ({
+  dispatch: jest.fn(),
+}))
+
+jest.mock('uiSrc/telemetry', () => {
+  const actual = jest.requireActual('uiSrc/telemetry')
+  return { ...actual, sendEventTelemetry: jest.fn() }
+})
+
+const mockDeleteInstancesAction = jest.fn(
+  (instances: Instance[], cb?: () => void) => ({
+    type: 'MOCK_DELETE',
+    payload: { instances, cb },
+  }),
+)
+const mockFetchEditedInstanceAction = jest.fn((instance: Instance) => ({
+  type: 'MOCK_FETCH_EDITED',
+  payload: instance,
+}))
+const mockSetEditedInstance = jest.fn((instance: Instance | null) => ({
+  type: 'MOCK_SET_EDITED',
+  payload: instance,
+}))
+
+jest.mock('uiSrc/slices/instances/instances', () => ({
+  deleteInstancesAction: (...args: any[]) =>
+    // @ts-expect-error
+    mockDeleteInstancesAction(...(args as any)),
+  fetchEditedInstanceAction: (...args: any[]) =>
+    // @ts-expect-error
+    mockFetchEditedInstanceAction(...(args as any)),
+  setEditedInstance: (...args: any[]) =>
+    // @ts-expect-error
+    mockSetEditedInstance(...(args as any)),
+}))
+
+const instance: Instance = {
```

Comment:
Note: If you want, you can export this one as a reusable factory, just like we already have such for the RDI Instances ([example](https://github.com/redis/RedisInsight/blob/056ce218d70c86ad92a780d43f8666ce7382569f/redisinsight/ui/src/mocks/rdi/RdiInstance.factory.ts))

Reply:
Lovely, it looks much better now!